re.c

```diff
@@ -2337,12 +2335,14 @@ match_named_captures(VALUE match)
 
 /*
  *  call-seq:
- *     mtch.string   -> str
+ *    string -> string
  *
- *  Returns a frozen copy of the string passed in to <code>match</code>.
+ *  Returns a frozen copy of the target string:
```

Comment:
This isn't accurate if the string passed in is already frozen (the existing documentation was wrong).  In that case, it returns the string passed in, not a frozen copy.

Reply:
Fixed.